web/src/main/java/pl/edu/agh/papaya/security/TokenRepository.java

```diff
@@ -0,0 +1,32 @@
+package pl.edu.agh.papaya.security;
+
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
+import java.time.Duration;
+import java.util.Optional;
+import java.util.UUID;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+@Component
+class TokenRepository {
+
+    private static final int TOKEN_CACHE_MAXIMUM_SIZE = 1000;
+    private static final Duration TOKEN_CACHE_EXPIRE_AFTER_ACCESS_DURATION = Duration.ofHours(2);
```

Comment:
Powiedziałabym, że `TOKEN_CACHE_EXPIRATION_TIME` mi się bardziej podoba, ale czytoważne

(Dwójek się akurat checkstyle nie czepia jeszcze :stuck_out_tongue:)

Reply:
Jeśli size ma być stałą to czemu expiration time nie? :thinking:

Reply:
Nie, nie, ma być, wszystko jest super. Tylko informuję, że checkstyle puszcza 0/1/2

Reply:
Wszystkie inne liczby już chyba są magiczne